docs/user/connected-accounts.rst

```diff
@@ -1,5 +1,5 @@
-Connecting your Git repository
-==============================
+How to connect your Git repository
+==================================
```

Comment:
This was intended as an explanation since there isn't anything actionable. I can guess at several reasons that a user should read the article, and since some of them are also how-to oriented, I don't see anything bad with the refactor.

We can call it a how-to, that can work as well :+1: